package.json

```diff
@@ -34,6 +34,7 @@
     "jest:e2e": "jest -c jest.config.e2e.js --",
     "coverage": "cross-env NODE_ENV=test LOG_LEVEL=silent MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --collectCoverage --",
     "jest-watch": "cross-env MOCK_REDIS=1 jest --watch --",
```

Comment:
Reading this, I notice we are missing `-c jest.config.js` in `jest-watch` too, if you want to add.